README.adoc

```diff
@@ -209,6 +210,11 @@ https://www.lucidchart.com/documents/edit/6293d791-8f70-47bb-8136-438128c27129/0
 
 === Using Swagger UI
 
+==== By browser
+
+The swagger api doc is generated in link:./api/swagger.yaml[/api/swagger.yaml] and you can check the REST API definition with this file by use the https://petstore.swagger.io/[Demo] tool online for swaggerUI or https://chrome.google.com/webstore/detail/swagger-ui-console/ljlmonadebogfjabhkppkoohjkjclfai?hl=en[Chrome extension].
```

Comment:
```suggestion
by using the
```

Reply:
@camilamacedo86 This conversation is resolved but it looks like the change is still the same?

Reply:
Sorry, you can check now.